borrowing_app/views.py

```diff
@@ -1,3 +1,18 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+
+from borrowing_app.models import Borrowing
+from borrowing_app.serializers import BorrowingSerializer, BorrowingListSerializer, BorrowingDetailSerializer
+
+
+class BorrowingViewSet(viewsets.ModelViewSet):
+    queryset = Borrowing.objects.all()
```

Comment:
won't you need a def get_querry_set() method in this case? borrowing model has many-to-many with books and many-to-one with user, you'll probably need to use select_related, prefetch_related so it doesn't create duplicated querys